website/docs/reference/references-overview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "About References"
+id: "references-overview"
+sidebar_label: "About References"
+description: "Connect dbt to any data platform in dbt Cloud or dbt Core, using a dedicated adapter plugin"
+hide_table_of_contents: true
+---
+
+The References section contains reference materials for developing in dbt Cloud and dbt Core. 
```

Comment:
One thing I've seen our teams do when a feature is applicable to both Core and Cloud is to use only `dbt`.  Might save you the trouble of having to type core and cloud repeatedly. I don't know if that's officially accepted branding, but might be worth talking about.